docs/intro/getting-started.md

```diff
@@ -6,39 +6,35 @@ sidebarDepth: 0
 
 ## Download and Install
 
-This step should be skipped if you purchased a myNode device!
+You can skip this step if you purchased a pre-loaded myNode device!
 
-Setting up a new myNode device can be done in a few easy steps. First, you need to download the myNode image for your device type and flash it on an SD card according to the instructions on the download page.
+Setting up a new myNode device can be done in a few simple steps. First, you need to download the myNode image for your device type and flash it on an SD card according to the instructions on the download page.
 
-## Power On and Attach HDD
+## Power On and Attach External Storage
 
 Next, power on the device and attach an external HDD or SSD.
 
-Note: The drive will be re-formatted for use by myNode and existing data will be lost!
+**Note**: The drive will be re-formatted for use by myNode and existing data will be lost!
 
 ## Connecting to myNode
 
-Once it's powered, visit `http://mynode.local/` or `http://mynode_ip_address/` in a web browser on your PC, laptop or phone. The myNode device runs a webserver, so to connect to it's GUI, you use a browser on a different device.
+Your myNode device runs its own local web server that you can access with any browser on the same network. Visit [’http://mynode.local/’](http://mynode.local/) or [’http://(mynode.IP.address)/’](http://mynode_ip_address/) in a web browser on a network-connected device. Here replace ’(mynode.IP.address)’ with the IP address of the myNode device.
```

Comment:
![image](https://user-images.githubusercontent.com/6998593/102578515-596ce600-4120-11eb-936a-1251173c34f8.png)

apostrophe look odd to me